common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4845,6 +4846,17 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val CLIENT_SHUFFLE_DYNAMIC_RESOURCE_FACTOR: ConfigEntry[Double] =
+    buildConf("celeborn.client.shuffle.dynamicResourceFactor")
+      .categories("client")
+      .version("0.6.0")
+      .doc("The ChangePartitionManager will check whether (unavailable workers / shuffle allocated workers) " +
```

Comment:
Change the doc to adjust the new implementation.

Reply:
Done~